tools/fuzzilli/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
We made the fuzzers directory so that you can place different types of fuzzers in a single directory.
Just move fuzzili.cpp there and add this cmake stuff to the existing cmakelists.txt file